metrics.go

```diff
@@ -21,21 +21,38 @@ var (
 
 func sanitizeValue(label string, value interface{}) string {
 
-	// Convert to a string, no matter what underlying type it is
-	var labelValue string
-	if value != nil {
-		labelValue = fmt.Sprintf("%v", value)
+	if value == nil || value == "" || value == "-" {
+		return ""
 	}
 
+	// Convert to a string, no matter what underlying type it is
+	labelValue := fmt.Sprintf("%v", value)
+	labelValue = strings.TrimSpace(labelValue)
+
 	switch label {
 	case "content_type":
-		labelValue = strings.Split(labelValue, ";")[0]
+		labelValue = strings.ToLower(labelValue)
+		if strings.HasPrefix(labelValue, "image") {
+			labelValue = "image"
+		} else if strings.HasPrefix(labelValue, "text/html") {
+			labelValue = "html"
+		} else if strings.HasPrefix(labelValue, "text/css") {
+			labelValue = "css"
+		} else if strings.Contains(labelValue, "javascript") {
+			labelValue = "javascript"
+		} else {
+			labelValue = "other"
+		}
 	case "hostname":
 		labelValue = strings.Split(labelValue, ":")[0]
+		labelValue = strings.ToLower(labelValue)
```

Comment:
Given [Pavel's findings](https://sectionio.slack.com/archives/C05200C3R/p1571465437038700), should we check for invalid characters in hostname and just discard the metric if present?